### PR TITLE
Update to TOS link & Translations

### PR DESCRIPTION
**Pull request by okariJC**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ However changes to the Plugin vs Local will be separated into difference Section
 
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unknown]
+### Plugin
+#### Changed
+ - Terms of Service Link now directs to https://www.jamcity.com/privacyandterms/
+    - Also made button bigger on both landscape and portrait
+ - Updted Localization
```
Comment (nickcashsgn) [inline CHANGELOG.md]: 🐃  - `Updated`

Comment (okariJC) [reply CHANGELOG.md]: Fixed in `Sync from VV` PR

File: Assets/Plugins/Jamcity/JamCityID/Scripts/TranslationSheetProcessor.cs [hunk 2]
```diff
@@ -159,7 +159,7 @@ private void CreateLanguageSheet(List<string> keys, List<string> forGameList, st
 
             var dictionary = new Dictionary<string, string>();
 
-            for (var rowIndex = 0; (rowIndex < keys.Count) && rowIndex < forGameList.Count; rowIndex++)
+            for (var rowIndex = 0; rowIndex < keys.Count && rowIndex < forGameList.Count && rowIndex < values.Count; rowIndex++)
```
Comment (chrissJC) [inline Assets/Plugins/Jamcity/JamCityID/Scripts/TranslationSheetProcessor.cs]: Some parentheses could make this easier to read.

Comment (okariJC) [reply Assets/Plugins/Jamcity/JamCityID/Scripts/TranslationSheetProcessor.cs]: Fixed in Translation refactor :D

**Pull request merged**